### DISCUSSION TITLE
how to create an empty project

### DISCUSSION THREAD

**Discussion by skybrian** [freshframework/fresh#2200] [Q&A] [answered]
The 'hello world' app created by 'deno run -A -r https://fresh.deno.dev' is nice enough, but it has a lot of demo content that's not appropriate when writing your own web app. (For example, it has the fresh logo on every page, a fresh favicon, and so on.)

I could delete that stuff manually, but I'm not sure I'll get all of it. I like to create a lot of little demo websites and I might miss something. Is there a simple way to create an *empty* project?

**Comment by marvinhagemeister** [comment, marked as answer]
I don't think we have a way to do that at the moment. The current way is to delete the files you don't need from the default template.
